superset/db_engine_specs/couchbase.py

```diff
@@ -0,0 +1,62 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from superset.db_engine_specs.base import BaseEngineSpec
+from superset.constants import TimeGrain
+from datetime import datetime
+import sqlparse
+from superset.db_engine_specs.base import BaseEngineSpec
+from superset.superset_typing import ResultSetColumnType, SQLAColumnType
+from flask_babel import gettext as __
+from superset.db_engine_specs.base import BaseEngineSpec
+from typing import (
+    Any
+)
+
+class CouchbaseEngineSpec(BaseEngineSpec):
```

Comment:
you can you add `BasicParametersMixin` to this since using MYSQL dialect
`https://github.com/apache/superset/blob/master/superset/db_engine_specs/base.py#L2276`